x-pack/plugins/cloud/server/plugin.ts

```diff
@@ -145,9 +155,11 @@ export class CloudPlugin implements Plugin<CloudSetup, CloudStart> {
     };
   }
 
-  public start() {
+  public start(): Pick<CloudSetup, 'isCloudEnabled' | 'projectsUrl' | 'baseUrl'> {
```

Comment:
__question:__ Any reason we don't want to extend `CloudStart`? I remember our auto-generated docs didn't like `Pick<...>` and we always opted to dedicated interfaces instead, but I'm not sure if it's still the case.

Reply:
Good catch, will change :)